Sources/Amplitude/AMPStorage.m

```diff
@@ -0,0 +1,103 @@
+//
+//  File.m
+//
+//
+//  Created by Dante Tam on 6/10/21.
+//
+
+#import <Foundation/Foundation.h>
+#import <Amplitude.h>
+#import "AMPStorage.h"
+
+@implementation AMPStorage
+
++ (NSString *)getAppStorageAmpDir {
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
+    NSString *path = [paths firstObject];
+    NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+    return [NSString stringWithFormat:@"%@/%@/%@", path, @"/", bundleIdentifier];
+}
+
++ (NSString *)getDefaultEventsFile {
+    NSString *baseDir = [AMPStorage getAppStorageAmpDir];
+    NSString *path = [baseDir stringByAppendingString:@"/amplitude_event_storage.txt"];
```

Comment:
Another question here. Originally, we have different databases for different instance names.  Should we also consider naming `.txt` file-related instance name here?

Reply:
Oh yeah, if we have multiple instances, we should use different files. Instance name should be passed as a parameter now